### PR TITLE
Add eventbus docs

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/tildagon-apps/widgets-and-hardware/eventbus.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+4. Register your event handler, for example in the `__init__` method of your app with the event and the event handler. Depending on whether the event handler is a synchronous or asynchronous method call `on()` or `on_async()`:
+
+    ```python
+    def __init__(self):
+        eventbus.on(SpecialEvent, self.handle_event, self.app)
+    ```
+
+    ```python
+    def __init__(self):
+        eventbus.on(SpecialEvent, self.handle_event_async, self.app)
```
Comment (b0xcat) [inline docs/tildagon-apps/widgets-and-hardware/eventbus.md:47]: I think this should be on_async right?

File: docs/tildagon-apps/widgets-and-hardware/eventbus.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    ```python
+    def update(self, delta):
+        # If something happens
+        eventbus.emit(SpecialEvent())
+    ```
+
+    ```python
+    def update(self, delta):
+        # If something happens
+        eventbus.emit_async(SpecialEvent())
```
Comment (b0xcat) [inline docs/tildagon-apps/widgets-and-hardware/eventbus.md:61]: And for async event emitting:
```suggestion
    async def update(self, delta):
        # If something happens
        await eventbus.emit_async(SpecialEvent())
```

File: docs/tildagon-apps/widgets-and-hardware/eventbus.md (hunk 1 above)
Comment (npentrel) [reply docs/tildagon-apps/widgets-and-hardware/eventbus.md:47]: yep

Comment (npentrel) [inline docs/tildagon-apps/widgets-and-hardware/eventbus.md:47]:
```suggestion
        eventbus. on_async(SpecialEvent, self.handle_event_async, self.app)
```

**Comment by npentrel** [comment]
fixed @b0xcat - can you or @hughrawlinson merge?

**Pull request merged**